src/libraries/System.Security.Cryptography.Encoding/tests/Cbor.Tests/CoseKeyHelpers.cs

```diff
@@ -0,0 +1,261 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Security.Cryptography;
+
+namespace System.Formats.Cbor.Tests
+{
+    public static class CborCoseKeyHelpers
+    {
+        public static byte[] ExportCosePublicKey(ECDsa ecDsa, HashAlgorithmName hashAlgName)
```

Comment:
Should this be called WritePublicKey? Cose is assumed for the class name. Other option perhaps leave the name Export but make this an extension method

Reply:
I was close to making this an extension but my understanding is that this should be avoided in test code. I'm assuming the plan here is that a variant of this will eventually find itself as an extension method in S.S.C.Algorithms once the CBOR apis become available.